src/workload.rs

```diff
@@ -277,6 +364,23 @@ impl xds::Handler<XdsWorkload> for Arc<Mutex<WorkloadStore>> {
     }
 }
 
+impl xds::Handler<XdsAddress> for Arc<Mutex<WorkloadStore>> {
+    fn handle(&self, updates: Vec<XdsUpdate<XdsAddress>>) -> Result<(), Vec<RejectedConfig>> {
+        let mut wli = self.lock().unwrap();
```

Comment:
**suggestion:** consider using `expect` instead of `unwrap` so it might be easier to locate the source if `lock()` returns an error variant and this panics.

Reply:
I think the panic will show the source file/line

Reply:
You're right John. It will show file and line. I prefer the expect message myself. I also believe it's considered "good form" for rust to not have unwrap in production code. If you want unwrap functionality use expect along with a message about why it's thought to never panic. Sometimes we're smarter than the compiler is and for that there's expect; for everything else there pattern matching.

Anyway, don't block on this. It's not a hill anyone need die on.

Reply:
`clippy::unwrap_used` could be added in CI at some point if we decide we want to remove all the unwraps from the codebase and prevent it from being introduced again